torch/_inductor/codegen/wrapper.py

```diff
@@ -2072,137 +2082,136 @@ def get_output_refs(self):
         ]
 
     def generate_return(self, output_refs):
-        if V.graph.aot_mode:
```

Comment:
This part is hard to review with eyeball. It's because of removing this checking caused an indentation difference.